dev-requirements.txt

```diff
@@ -9,6 +9,7 @@ pylint==2.6.0
 pytest==7.1.2
 pytest-cov==3.0.0
 requests-mock==1.9.3
+setuptools>=64.0.2
```

Comment:
should we more strictly pin this to avoid a similar situation in the future?

Reply:
I considered this, but we were getting it by default from pip before and I didn't know if we would start causing version conflicts with pip if we do that.

Reply:
good call, hopefully this should be a very rare occurence!